src/Shopsys/ShopBundle/Resources/translations/messages.de.po

```diff
@@ -2,884 +2,899 @@ msgid ""
 msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Language: en\n"
+"Language: de\n"
 
 msgid "(If I buy on the company behalf)"
-msgstr "(de) (If I buy on the company behalf)"
+msgstr "(Wenn ich im Namen des Unternehmens kaufe)"
 
 msgid "(Products will be delivered to this address)"
-msgstr "(de) (Products will be delivered to this address)"
+msgstr "(Die Produkte werden an diese Adresse geliefert)"
 
 msgid "(This address will be on the tax invoice)"
-msgstr "(de) (This address will be on the tax invoice)"
+msgstr "(Diese Adresse befindet sich auf der Steuerrechnung)"
 
 msgid "100% Satisfaction"
-msgstr "(de) 100% Satisfaction"
+msgstr "100% Zufriedenheit"
 
 msgid "24/7 Help Center"
-msgstr "(de) 24/7 Help Center"
+msgstr "24/7 Hilfe-Center"
 
 msgid "Accessories"
-msgstr "(de) Accessories"
+msgstr "Zubehör"
+
+msgid "Account"
+msgstr ""
```

Comment:
The translation from https://github.com/shopsys/demoshop/pull/71#issuecomment-528836928 is missing...
```suggestion
msgstr "Benutzerkonto"
```